dance/transforms/normalize.py

```diff
@@ -206,11 +206,11 @@ def __init__(
         self,
         split_names: Optional[Union[Literal["ALL"], List[str]]] = None,
         batch_key: Optional[str] = None,
-        min_cells: int = 5,
+        min_cells: int = 2,
         gmean_eps: int = 1,
-        n_genes: Optional[int] = 2000,
+        n_genes: Optional[int] = 3000,
         n_cells: Optional[int] = None,
-        bin_size: int = 500,
+        bin_size: int = 200,
```

Comment:
Are the purpose of these changes to the default setting to be consistent with the original implementations? In practice, we must avoid changes like this as it easily breaks user compatibility in the sense that whenever they upgrade the dance version, artifacts like these will "silently" change their results.

For now this is fine, because we have not released this in the latest version yet. But please keep in mind that once this is released in a stable version, we should be very careful about updating default options. If we have to, we must give explicit user warnings.